src/app/forms/templates/psv/psv-tech-record.template.ts

```diff
@@ -61,15 +68,14 @@ export const PsvTechRecord: FormNode = {
       children: [
         {
           name: '0',
-          label: 'Axel',
           value: '',
           type: FormNodeTypes.GROUP,
           children: [
             {
               name: 'parkingBrakeMrk',
-              label: 'Axles fitted with a parking brake',
```

Comment:
I think we might need some guidance with this one as the ticket says it's an optional boolean but the backend is saying it's a required field, so we might need to update the backend in a similar way we did for dda ceritifcateIssued. I think it'd be good to speak with service design though about what the options are in this dropdown as it seems confusing from the ticket how it would work
![image](https://user-images.githubusercontent.com/92087051/196953490-b54ad227-d9d2-44f6-a9ce-5f400379d334.png)